scripts/add_secret_in_keyvault.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
scripts/create_password_in_keyvault.sh has very similar functionality, would it be better to modify this script and add an optional new_secret argument (so not be break existing scripts) instead of creating a new script?